.authors.yml

```diff
@@ -0,0 +1,9 @@
+[{name: Christopher J. Wright, email: cjwright4242gh@gmail.com, num_commits: 3, first_commit: 2017-07-25
```

Comment:
Hmm this should have nicer YAML formatting.  Not sure why it came out like this...

Reply:
Also you need to add "christopher" to your aliases list to get the mailmap to generate correctly.